src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -165,19 +175,33 @@ export function DashboardTopNav({ embedSettings, redirectTo }: DashboardTopNavPr
       // set only the dashboardTitleBreadcrumbs because the main breadcrumbs automatically come as part of the navigation config
       serverless.setBreadcrumbs(dashboardTitleBreadcrumbs);
     } else {
-      // non-serverless regular breadcrumbs
-      setBreadcrumbs([
-        {
-          text: getDashboardBreadcrumb(),
-          'data-test-subj': 'dashboardListingBreadcrumb',
-          onClick: () => {
-            redirectTo({ destination: 'listing' });
+      /**
+       * non-serverless regular breadcrumbs
+       * Dashboard embedded in other plugins (e.g. SecuritySolution)
+       * will have custom leading breadcrumbs for back to their app.
+       **/
+      setBreadcrumbs(
+        customLeadingBreadCrumbs.concat([
```

Comment:
Before - dashboard breadcrumbs in Security Solution was incorrect:
![incorrect_breadcrumbs](https://github.com/elastic/kibana/assets/6295984/16195206-adba-4cb4-beec-86d8a5b09d09)



After:
Traditional:
<img width="524" alt="dashboard_breadcrumbs_ess" src="https://github.com/elastic/kibana/assets/6295984/f46cf9d4-170a-452b-991f-c96cc5d26115">

Serverless:
<img width="577" alt="dashboard_breadcrumbs_serverless" src="https://github.com/elastic/kibana/assets/6295984/6369fcc0-eec8-432a-b930-f590ade4f8f5">